open_discussions_api/channels/client.py

```diff
@@ -0,0 +1,27 @@
+"""Channels API"""
+import json
+
+from open_discussions_api.base import BaseApi
+
+from open_discussions_api.channels.constants import CHANNEL_ATTRIBUTES, VALID_CHANNEL_TYPES
+
+
+class ChannelsApi(BaseApi):
+    """Channels API"""
+
+    def create(self, **channel):
```

Comment:
nit: this variable name is misleading: I think it should be something like `channel_args`, `channel_params` or whatever indicates that this is a data structure with multiple entries

Reply:
sure